### PR TITLE
feat: Provide an enhanced readme

### PR DESCRIPTION
**Pull request by Sceat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,72 @@
...
-
-### v2
-
-The V2 is a unreleased version of AresRPG, the goal of this version is to have an procedural open-world instead
-of a floor system and more classes than the v1
-
-## Requirements
+<h1 align=center>AresRPG</h1>
+<p align=center>
+  <img src="https://i.imgur.com/5doezIg.png" />
```
Comment (DeltaEvo) [inline README.md:3]: Keep the banner or upload this file to the repository

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

**Comment by DeltaEvo** [comment]
Thanks for the changes this is much better :smile: 

Can you also rename your commit to Capitalize the first letter ?

**Review by DeltaEvo — CHANGES_REQUESTED** [review]
Last little thing can you merge/squash your two commits together ?

**Comment by Sceat** [comment]
🤬

**Review by DeltaEvo — APPROVED** [review]

**Pull request merged**